python/py-pyaudio/Portfile

```diff
@@ -6,6 +6,7 @@ PortGroup               python 1.0
 name                    py-pyaudio
 set my_name             PyAudio
 version                 0.2.11
+revision                1
```

Comment:
This is not necessary since there should be no change to any existing build of the software since the code is not changing, the build is not changing, and the libraries depended on should not be changing.